### PR TITLE
Update seeds.rb with sample user

### PR DESCRIPTION
**Pull request by kiliczsh**
When the application is installed it is nice to have a default user to get onboard quickly. 

```bash
rails db:seed
```

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -7,3 +7,10 @@
 #   ["Action", "Comedy", "Drama", "Horror"].each do |genre_name|
 #     MovieGenre.find_or_create_by!(name: genre_name)
 #   end
+
+# Create the default user
+family = Family.create(name: "The Smiths")
```
Comment (sebastiancaraballo) [inline db/seeds.rb:12]:
```suggestion
family = Family.create_or_find_by!(name: "The Smiths")
```
Maybe: https://apidock.com/rails/ActiveRecord/Relation/create_or_find_by

**Review by robzolkos — APPROVED** [review]
Thank you! This is much needed!  Should we put something in the readme to guide people to its existence?

**Comment by robzolkos** [comment]
> db:setup has seed by default

True - but unless a user goes digging into the seed.rb file they won't know how to login to the app.   Normally I've seen instructions like 

run `db:setup` - this will setup a test user with login credentials ...... and password .....

File: db/seeds.rb [hunk 2]
```diff
@@ -7,3 +7,10 @@
 #   ["Action", "Comedy", "Drama", "Horror"].each do |genre_name|
 #     MovieGenre.find_or_create_by!(name: genre_name)
 #   end
+
+# Create the default user
+family = Family.create_or_find_by!(name: "The Smiths")
+puts "Family created: #{family.name}"
+user = User.create(first_name: "John", last_name: "Smith", email: "john@smith.com",
```
Comment (sebastiancaraballo) [inline db/seeds.rb:14]:
```suggestion
user = User.create_or_find_by!(first_name: "John", last_name: "Smith", email: "john@smith.com",
```
same here

**Review by sebastiancaraballo — APPROVED** [review]
Nice

**Comment by kiliczsh** [comment]
True idea, I need a little bit time to update it.

**Comment by Shpigford** [comment]
> > db:setup has seed by default
> 
> True - but unless a user goes digging into the seed.rb file they won't know how to login to the app. Normally I've seen instructions like
> 
> run `db:setup` - this will setup a test user with login credentials ...... and password .....

I agree that adding a line to the README would be helpful.

**Pull request merged**